style.md

```diff
@@ -21,6 +21,22 @@ The redundancy reduction from removing the namespacing prefix is not useful enou
 Because in many cases the Go default for the type is a no-op in the spec (sources [here][no-pointer-for-strings], [here][no-pointer-for-slices], and [here][no-pointer-for-boolean]).
 The exceptions are entries where we need to distinguish between “not set” and “set to the Go default for that type” ([source][pointer-when-updates-require-changes]), and this decision should be made on a per-setting case.
 
+## Links
+
+Internal links should be [relative links][markdown-relative-links] when linking to content within the repository.
+Internal links should be used inline.
+
+External links should be collected at the bottom of a markdown file and used as referenced links.
+See 'Referenced Links' in this [markdown quick reference][markdown-quick-reference].
+The use of referenced links in the markdown body helps to keep files clean and organized.
+This also facilitates updates of external link targets on a per-file basis.
+
+Referenced links should be kept in two alphabetically sorted sets, a general reference section followed by a man page section.
+To keep Pandoc happy, duplicate naming of links within the scope of the repository should be avoided by appending an '_N' to the link tagname, where 'N' is some number not currently in use.
```

Comment:
“within the scope of the repository” → “within [pages compiled into the spec][1]”.  Although keeping that link up to date might be hard, and we may want to say “within pages listed in the Makefile's `DOC_FILES`”.

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc4/Makefile#L21-L32